brave/brave-core
# (Wallet) Tweak onboarding layouts and styles following updated Figma design

## Resolves https://github.com/brave/brave-browser/issues/38890
## Resolves https://github.com/brave/brave-browser/issues/38080
## Resolves https://github.com/brave/brave-browser/issues/38912


Improves text styles adopting imported attributes.
Implements new checkbox colors for selected, unselected and disabled states.
Improves button colors in dark mode.
Sets checkboxes for Solana and Ethereum networks as disabled in network selection screen during onboarding. 



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clear Wallet data, if any
- Navigate to Wallet section to create a new Wallet
- Navigate to network selection screen
- Observe Solana and Ethereum cannot be deselected and the checkbox color is grayed out
